tests/ui/pub_use.rs

```diff
@@ -0,0 +1,8 @@
+#![warn(clippy::pub_use)]
+#![allow(unused_imports)]
+#![no_main]
+
+// should be linted
+pub use std::fmt;
+// should not linted
+use std::env;
```

Comment:
Can the test like the example be added?

Reply:
It was like the example if I recall correctly, I think the rebase changed it back. Will update it

Reply:
Restored